ow_dynamic_terrain/msg/scoop_dig_phase.msg

```diff
@@ -0,0 +1,12 @@
+std_msgs/Header header
+bool digging        # returns true when scoop appears to be performing a
+                    # digging motion
+uint8 phase         # returns an enumerate that represents what phase of a dig
+                    # the scoop is estimated to be in
+
+# phase enumeration constants
+uint8 NOT_DIGGING = 0
+uint8 SINKING     = 1
+uint8 PLOWING     = 2
+uint8 RETRACTING  = 3
+
```

Comment:
Would it be useful to have a 5th state for when the dig is complete but the scoop has not yet been emptied (Discard or Deliver)?  This could be further refined to distinguish when regolith was collected or not.

Reply:
It could be done. Who/what would need to know that information?

Reply:
Autonomy, but I don't have a specific use case in mind.  It was just a thought.  This can be a future addition if there's a need.

Reply:
Sounds good. Something to consider.